CIPs/cip-TBC.md

```diff
@@ -0,0 +1,91 @@
+---
+cip: TBC
```

Comment:
please add the number 25

Reply:
Also please modify the file number from TBC to 0025 

Thank you so much for contributing this CIP!